examples/tutorials/An_Introduction_to_Polymers_and_Their__Representation.ipynb

```diff
@@ -0,0 +1,1465 @@
+{
```

Comment:
<p>Add the discord, and bibtex citation. refer to other tutorials on deepchem.io for the format</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3930/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

Reply:
Will do ! Thank you for your review.